source/gui/__init__.py

```diff
@@ -494,17 +494,17 @@ def __init__(self, frame: MainFrame):
 		menu_tools = self.toolsMenu = wx.Menu()
 		if not globalVars.appArgs.secure:
 			# Translators: The label for the menu item to open NVDA Log Viewer.
-			item = menu_tools.Append(wx.ID_ANY, _("View log"))
+			item = menu_tools.Append(wx.ID_ANY, _("&View log"))

```

Comment:
Perhaps this should be changed to `&l`? It's a bit more intuitive

Reply:
```suggestion
			item = menu_tools.Append(wx.ID_ANY, _("View &log"))
```

Reply:
In my idea, I have only made explicit shortcuts tha were previously implicit when possible. Since "V" was used previously as an implicit shortcut, I have kept it.

Your suggestion makes sense, but I do not know if there are training material (from NV Access or others) which indicate NVDA+N, T, V to open the log viewer. I would like to have @Qchristensen's opinion about it before accepting the suggestion.

Reply:
I don't have a strong preference either way here - I don't think I've written any instructions (not recently at least, and not in say Basic Training for NVDA - I mention the log viewer and where it is, but don't go through the specific keys to press to open it).  If there are commonly used community instructions which mention it, as @XLTechie notes is the case with the COM Registration fixing tool in the conversation further down, then I'd be happy to follow their lead, but otherwise, I'm happy either way here - there is some logic to L, given the first three options (view log, braille viewer and speech viewer, all relate to viewing something so without context it isn't immediately obvious which one V would be for.